sql/src/main/java/com/vmware/ddlog/DDlogJooqProvider.java

```diff
@@ -164,14 +180,22 @@ public DDlogJooqProvider(final DDlogAPI dDlogAPI, final List<String> sqlStatemen
 
     private void rollback() {
         try {
+            if (trace)
+                System.out.println("Rolling back transaction");
             dDlogAPI.transactionRollback();
+            if (trace)
+                System.out.println("Transaction rolled back");
         } catch (final DDlogException e) {
+            if (trace)
```

Comment:
Okay for now but we should be using sl4j instead of all the if(trace) -> stdout stuff.

Reply:
I don't know if logging makes sense for testing.
I actually used this to debug a nasty bug where the exception was not caught here and the next test would fail because the started transaction was not rolledback.